### DISCUSSION TITLE
What is the correct way to pronounce "Mantine"?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#675] [Q&A] [answered]
There is some debate about the pronounciation of "Mantine".

Looks like [according to Pokemon community](https://www.youtube.com/watch?v=KvzMQ3NkL9g) the correct way to say it is "Man-TEEN", not "Man-TINE".

@rtivital What is the official way to pronounce it? :)

**Comment by rtivital** [comment, marked as answer]
I call it `mɑ:nti:n` – https://www.londonschool.com/blog/phonetic-alphabet/

**Reply by ryami333** [reply]
For anyone who was confused by this link (like I was) and you're looking for a quick answer: it rhymes with "team", not "time".

**Comment by danols** [comment]
Does this help, https://www.google.com/search?q=mantine+pronunciation?
